docs/platform-integration/appmodel/permissions.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Permissions"
+description: "Learn how to use the .NET MAUI Permissions class, to check and request permissions. This class is in the Microsoft.Maui.ApplicationModel namespace."
+ms.date: 05/09/2022
+no-loc: ["Microsoft.Maui", "Microsoft.Maui.ApplicationModel"]
+---
+
+# Permissions
+
+This article describes how you can use the .NET Multi-platform App UI (.NET MAUI) `Permissions` class. This class allows you to check and request permissions at run-time. The `Permissions` type is available in the `Microsoft.Maui.ApplicationModel` namespace.
+
+[!INCLUDE [docs under construction](~/includes/preview-note.md)]
+
+## Get started
+
+The following information applies to Android platforms.
+
+[!INCLUDE [android-permissions](../includes/android-permissions.md)]
+
+## Available permissions
+
+.NET MAUI attempts to abstract as many permissions as possible. However, each operating system has a different set of permissions. Even though the API allows access to a common permission, there may be differences between operating systems related to that permission. The following table describes the available permissions:
+
+The following table uses ✔️ to indicate that the permission is supported and ❌ to indicate the permission isn't supported or isn't required:
+
+| Permission        | Android | iOS | UWP | watchOS | tvOS | Tizen |
```

Comment:
We no longer support watchOS... we should remove that from this... i would probably also remove Tizen and change UWP to Windows